src/VisualStudio/Core/Def/ColorSchemes/VisualStudio2019.xml

```diff
@@ -885,6 +912,9 @@
             <Color Name="xml literal - text">
                 <Foreground Type="CT_SYSCOLOR" Source="00000008" />
             </Color>
+            <Color Name="roslyn test code markdown">
```

Comment:
Should probably define a value for 'roslyn test code' even if it just matches the editor background. Otherwise, it would use the value set in the Format Definition constructor.

Reply:
gotcha.  how would i match the editor background?

Reply:
alternatively, i could remove the value in the FormatDefinition.

Reply:
I would steal the value Razor uses for HighContrast which is
`<Background Type="CT_SYSCOLOR" Source="00000005" />`

Reply:
so *everything* else is 08.  so i feel like in this case we should stick with that.

Reply:
Sorry, I am referring to the "roslyn test code" background classification which does not have an entry.

Reply:
![image](https://github.com/dotnet/roslyn/assets/611219/45dc3604-d52d-40e5-aad6-963f21c17651)